.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+# Description
+
+<!-- Please include a summary of the change. Please also include relevant motivation and context. -->
+
+<!-- Please link the issue which will be fixed (if any) here: -->
+Fixes # (issue)
+
+## Type of change
+
+<!-- Please delete options that are not relevant. -->
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
```

Comment:
I would also add:

[  ] Improvement (non-breaking change that does improve existing functionality)

For example, when refactoring code etc.

Reply:
Thanks, added!